angular.json

```diff
@@ -22,7 +22,10 @@
             "main": "src/test.ts",
             "tsConfig": "tsconfig.spec.json",
             "karmaConfig": "karma.conf.js",
-            "codeCoverage": true
+            "codeCoverage": true,
+            "styles": [
+              "src/css/archwizard.scss"
```

Comment:
With this PR we basically decouple the styles from the wizard components.
Therefore I'm asking myself if we can decouple the tests too? Is it possible to test the following separately:
- the wizard functionality
- the wizard design provided by the attached stylesheets

What do you think?

Reply:
I think yes, it is possible, but it will be rather awkward.
- For starters, the styles are configured globally in `angular.json`, so you will need to have two test configurations (one with `styles` and the other without) and run `ng test` twice with different configurations.
- Also, you will need to separate `*.spec.ts` files into two groups depending on the configuration under which they should be executed (`*.spec.ts` and `*.style-spec.ts`?).

I don't see a good purpose for this complication. Including the CSS/SCSS file shipped with angular-archwizard is required for the wizard to look as intended. So the only supported real-world scenario is that JS and CSS are included together. I think it is fine to only test the wizard in this configuration. And, if a user does not include the library's CSS files and provides their own instead, it is fine to consider it their responsibility if the wizard breaks.

Archwizard could provide two CSS files - one for layout (required) and another for basic styling (optional). In this case yes, perhaps it would be nice to test the wizard with and without the basic styles.

Reply:
The reason for this comment are the failing tests at https://github.com/madoar/angular-archwizard/runs/89348510. The problem is, that the tests depend on the styles to be applied. Therefore the intended wizard functionality is not anymore fulfilled if the user changes the styles. This seems strange to me.

Reply:
In that case, it is only a result of "improper" checks in the tests. If our goal is to isolate those tests from wizard, we should make assertions on `.textContent` rather than `.innerText`. The latter takes styles into account, which in this case is inappropriate (see the warning on https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText).

Reply:
I agree, we should change the tests to test the model and not the styles.

Reply:
Changed .innerText to .textContent in 9c578b3